docs/commands/respect.md

```diff
@@ -0,0 +1,163 @@
+
+---
+
+- -w, --workflow
+- [string]
+- Workflow names from the test file to run.
+  For example, the following command runs "first-flow" and "second-flow" workflows from the "test-file.yaml" test file: `npx @redocly/cli respect test-file.yaml --workflow first-flow second-flow`
+  {% admonition type="warning" %}
+  The `--workflow` option can't be used with `--skip`.
+  {% /admonition %}
+
+---
+
+- -s, --skip
+- [string]
+- Workflow names from the test file to skip.
+  For example, the following command skips the "first-flow" workflow from the "test-file.yaml" test file: `npx @redocly/cli respect test-file.yaml --skip first-flow`
+  {% admonition type="warning" %}
+  Warning: the `--skip` option can't be used with `--workflow`.
+  {% /admonition %}
+
+---
+
+- -v, --verbose
+- boolean
+- Runs the command in verbose mode to help with troubleshooting issues.
+  For example, the following command runs all workflows from the "test-file.yaml" test file in verbose mode: `npx @redocly/cli respect test-file.yaml --verbose`
+
+---
+
+- --har-output
+- string
+- Path for the `har` file for saving logs.
+  For example, the following command runs all workflows from the "test-file.yaml" test file and saves the logs to the "logs.har" file: `npx @redocly/cli respect test-file.yaml --har-output='logs.har'`
+
+---
+
+- --json-output
+- string
+- Path for the 'json`file for saving logs. For example, the following command runs all workflows from the "test-file.yaml" test file and saves the logs to the "logs.json" file:`npx @redocly/cli respect test-file.yaml --json-output='logs.json'`
+
+---
+
+- --input
+- string
+- Input parameters with values that are mapped to the workflow inputs description.
+  For example, the following command maps the "userEmail" and "userPassword" inputs and values to all workflows in the "test.yaml" test file: `npx @redocly/cli respect test.yaml --input userEmail=name@redocly.com --input userPassword=12345`.
+  You can also use an environment variable to set the input, as in the following example: `REDOCLY_CLI_RESPECT_INPUT='userEmail=name@redocly.com,userPassword=12345' npm run cli respect test.yaml`
+
+  You can even include nested values, as in the following example command that maps the "nestedKey" input and value to all workflows in the "test-file.yaml" test file: `npx @redocly/cli respect test-file.yaml --input '{"key": "value", "nested": {"nestedKey": "nestedValue"}}'`.
+  You can also use an environment variable to set the input, as in the following example: `REDOCLY_CLI_RESPECT_INPUT='{"key":"value","nested":{"nestedKey":"nestedValue"}}' npx @redocly/cli respect test-file.yaml`
+
+---
+
+- --server
+- string
+- Server overrides for the `sourceDescriptions` object.
+  For example, the following command runs all workflows from the "test-file.yaml" test file and instead of using the server listed in the API description, uses the server at "https://test.com": `npx @redocly/cli respect test-file.yaml --server test=https://test.com`
```

Comment:
```suggestion
  For example, the following command runs all workflows from the "test-file.yaml" Arazzo description and instead of using the server listed in the API description, uses the server at "https://test.com": `npx @redocly/cli respect test-file.yaml --server test=https://test.com`
```